### PR TITLE
Implement total USD asset balance

### LINKED ISSUE #77: implement total usd value for all assets combined
Screenshot 2023-09-24 at 2 24 28 PM

### PR DESCRIPTION
**Pull request by Sirmorrison**
Fix #77

### CODE REVIEW

**Comment by ukane-philemon** [comment]
Did a little test and noticed:
1. The assets card does not update the dollar val display when the rate source is disabled from the settings page but it does when it's enabled.
2. I think this is the PR that should remove the little toggle at the top as per the discussion on matrix.

**Comment by dreacot** [comment]
please resolve branch conflicts

**Comment by dreacot** [comment]
on testing the functionality i noticed there might have been a miscommunication with the way this should work, the total USD Balance, works with the `Fetch Exchange Rate` settings, the toggle might have been confusing

> I think this is the PR that should remove the little toggle at the top as per the discussion on matrix.

@ukane-philemon is right, remove the toggle, if rate fetching is enabled from the settings page, then it's enabled everywhere including the total usd balance, if it's disabled in settings, then it's disabled everywhere

also the text should change to "Total USD value"

Screenshot from 2023-09-29 10-36-05

Also please update the info text to say that this feature can be enabled or disabled from the app settings
Screenshot from 2023-09-29 10-40-18

File: ui/cryptomaterial/image.go [hunk 1]
```diff
@@ -57,7 +57,7 @@ func (img *Image) LayoutSize(gtx C, size unit.Dp) D {
 }
 
 func (img *Image) LayoutSize2(gtx C, width, height unit.Dp) D {
-	dst := image.NewRGBA(image.Rectangle{Max: image.Point{X: int(width * 2), Y: int(height * 2)}})
+	dst := image.NewRGBA(image.Rectangle{Max: image.Point{X: int(width), Y: int(height * 2)}})
```
Comment (dreacot) [inline ui/cryptomaterial/image.go:60]: This reintroduces this issue

Screenshot from 2023-09-29 10-44-42

**Review by dreacot — CHANGES_REQUESTED** [review]

**Review by dreacot — CHANGES_REQUESTED** [review]
Looks good overall!

hiding my balance doesn't persist when i close and reopen the app

Screenshot from 2023-10-02 10-34-47

**Review by dreacot — APPROVED** [review]

**Pull request merged**